faqs/galaxy/Datasets_ Data_Privacy_Features.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Data Privacy Features 
+area: History
+box_type: tip
+layout: faq
+contributors: [desmond-jasper]
+---
+
+Privacy controls are only enabled if desired. Otherwise, datasets remain public but unlisted. 
+This means that a dataset you've created is virtually invisible until you publish a link to it. 
+
+below are steps to setting private Histories
+
+1. Changing the privacy settings of individual dataset.
+-  click on the **dataset** name for a dropdown.
```

Comment:
The steps are more options than steps we would follow one after the other. This should be reflected here to avoid misleading users.

Reply:
@annefou, @beatrizserrano  thanks for your review, i will make all necessary correction immediately

Reply:
> @desmond-jasper please pay attention to the filename, it can't contain spaces (you can look at the filenames of the other FAQs)

oh, oh